common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -25,94 +14,68 @@ function Log-Warning($message) {
     }
 }
 
-function GetAllPackages {
-    $packages = Find-Package -Source $PACKAGE_FEED_URL -AllVersion -AllowPrereleaseVersions
-    if ($Daily) {
-        return $packages | Where-Object { $_.Version.Contains("alpha") }
-    }
-    return $packages | Where-Object { !$_.Version.Contains("alpha") -and !$_.Version.Contains("dev") }
+function Log-Info($message) {
+    Write-Host $message
 }
 
-function GetLatestPackage([array]$packageList, [string]$packageName) {
-    $versions = ($packageList
-        | Where-Object { $_.Name -eq $packageName }
-        | Select-Object -ExpandProperty Version)
+function AddPackageReference($csproj, $referenceNode, $packageInfo) {
+    $package = $packageInfo.Name
+    $version = $packageInfo.Version
 
-    if (!$versions) {
-        Write-Warning "Did not find any versions for $($packageName)"
-        return
+    if ($PreferDevVersion -and $packageInfo.DevVersion) {
```

Comment:
I don't think we are ever going to have a DevVersion set in this context. @ckairen we could actually use this value to store the version you setup with your work.

Reply:
This was based off of the original version of the packages script that @ckairen had put together; once we've got a better understanding of the new approach to read packages/versions, we can adjust the parameters.